Resources/Textures/SimpleStation14/Objects/Weapons/Guns/Pistols/mk45c.rsi/meta.json

```diff
@@ -0,0 +1,76 @@
+{
+    "version": 1,
+    "license": "CC-BY-SA-3.0",
+    "copyright": "ShockOmega",
```

Comment:
```suggestion
    "copyright": "DSC@ShockOmega#2505 (297567991112007681)",
```

Reply:
I have changed the copyright in the json file. It is now my discord  user+tag and my ID as suggested. Thanks!